contracts/truefi2/TrueFiPool2.sol

```diff
@@ -474,4 +486,13 @@ contract TrueFiPool2 is ITrueFiPool2, ERC20, Claimable {
 
         return mintedAmount;
     }
+
+    /**
+     * @dev Decrease provided amount percentwise by error
+     * @param amount Amount to decrease
+     * @return Calculated value
+     */
+    function withToleratedError(uint256 amount) internal view returns (uint256) {
+        return amount.mul(10000 - toleratedError * 100).div(10000);
```

Comment:
Just make `toleratedError` constant and make it equal 98, we'll get something like
        `return amount.mul(TOLERATED_STRATEGY_ERROR).div(100);`